markdown/usage/data_guidelines.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Resource Estimation and Data sharing
+subtitle:
+---
+
+## Data management
+Research data management concerns the organization, storage, preservation, and sharing of data that is collected or analyzed during a research project. Proper planning and data management facilitates sharing and allows others to validate and reuse the data. Also, funding agencies are recognizing the importance of research data management and some now request detailed Data Management Plans (DMP) as part of the grant application. Guidance is provided below to aid the creation of DMPs, estimate resources needed by nf-core workflows, and how to share the resulting data.
```

Comment:
Would it be ok to start with the motivation, instead of the definition? 

```suggestion
Funding agencies are recognizing the importance of research data management and some now request detailed Data Management Plans (DMP) as part of the grant application. Research data management concerns the organization, storage, preservation, and sharing of data that is collected or analyzed during a research project. Proper planning and data management facilitates sharing and allows others to validate and reuse the data. Guidance is provided below to aid the creation of DMPs, estimate resources needed by nf-core workflows, and how to share the resulting data.
```

(Github has a crazy amount of features. Time to get used to them.)

Reply:
Sure, starting with the definition sentence might scare some ppl away :)